frappe/public/js/frappe/social/components/EventCard.vue

```diff
@@ -7,4 +7,28 @@
 export default {
 	props: ['event'],
 }
-</script>
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
Please do not push conflicts

One suggestion.. keep updating your branch regularly to avoid conflicts